scripts/run_monitoring.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+#  Raymond Kirk (Tunstill) Copyright (c) 2021
+#  Email: ray.tunstill@gmail.com
+
+from __future__ import absolute_import, division, print_function
+
+import rospy
+
+from topic_store.file_parsers import resolve_scenario_yaml
+from topic_store.scenario import ScenarioMonitor
+
+
+def run_monitoring():
+    rospy.init_node("monitoring_runner_node", anonymous=True)
+
+    # Get parameters
+    scenario_file = resolve_scenario_yaml(rospy.get_param('~scenario_file', "default_config.yaml"))
+    verbose = rospy.get_param('~verbose', True)
```

Comment:
Just a suggestion, but it might be best to stick with the [ros verbose levels](https://wiki.ros.org/roscpp/Overview/Logging#Setting_Verbosity_Levels) rather than using a boolean, as this gives you more flexibility in the future

Reply:
I was looking at this but I found a thread where the ros logging was slowing down subscriptions to topics and the callback execution queue, so early on I decided to avoid all together and publish the logs/use print. Maybe it's time to revisit and see if it actually does slow it down!